sled-agent/src/instance_manager.rs

```diff
@@ -775,6 +800,53 @@ impl InstanceManagerRunner {
         tx.send(Ok(state)).map_err(|_| Error::FailedSendClientClosed)?;
         Ok(())
     }
+
+    async fn only_use_disks(
```

Comment:
It took me a minute to figure out what this name means --- this is saying "only use these disks that I'm telling you about, and not any other disks", right? I can't immediately think of a better name for this, though...

Reply:
Yeah, that's exactly what it means.

(Not sarcastically) If I add `please_only_use_disks` does it make it more clear?

Reply:
To be honest, I think even "use_only_these_disks" would make it a lot clearer!